submissions/ses3332002/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,81 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+class Animal {
+  constructor (name, friends, gender, saying, species) {
+    this.name = name;
+    this.friends = friends;
+    this.gender = gender;
+    this.saying = saying;
+    this.species = species;
+    this.legs = 4;
+    this.hands = 0;
```

Comment:
Not all animals in this world have hands by natural design.
Not they just got zero hands, but rather do not have any at all.
The beaty of OOP is that every base class owns and handles only props shared by its descendants, and any child class introduces and handles only properties unique to it and its descendants (in latter case it is also a base class in its turn).

Reply:
I remade it.